### PR TITLE
Material ui button added and are align horizontally

### PR DESCRIPTION
**Pull request by irfen-mi**
Hey!
Material ui button added and are align horizontally. i colored these buttons to orange which looks good you can just change the color what you want.
--Output

a

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,41 +1,66 @@
...
+              variant="contained"
+                sx={{ color: 'white', backgroundColor: 'orange' }}
+            >
+              <Link to="/about">About</Link>
+            </Button>
+          </li>
+          <li>
+            <Button
+              variant="contained"
+              startIcon={<ShoppingBasketIcon style={{color:"blue"}} />}
```
Comment (kamalbuilds) [inline src/components/Navbar.js:33]: This shopping basket icon is not getting rendered right?

File: package.json [hunk 2]
```diff
@@ -3,6 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.0",
+    "@emotion/styled": "^11.10.0",
+    "@material-ui/core": "^4.12.4",
+    "@mui/icons-material": "^5.8.4",
+    "@mui/material": "^5.10.1",
+    "@mui/styled-engine-sc": "^5.10.1",
```
Comment (kamalbuilds) [inline package.json:11]: is this needed ?

**Review by kamalbuilds — CHANGES_REQUESTED** [review]
good work @developer-irfan  just some more changes to look into .

**Comment by kamalbuilds** [comment]
This is the error that it leads to 
image

If you could resolve this it would work :+1:

File: src/components/Navbar.js (hunk 1 above)
Comment (irfen-mi) [reply src/components/Navbar.js:33]: @legendarykamal its material ui icon and properly render in my system.

**Comment by irfen-mi** [comment]
> This is the error that it leads to image
> 
> If you could resolve this it would work 👍

just install the react/jsx-runtime the error will be gone . 
open cmd and type npm install react/jsx-runtime end enter.

File: package.json (hunk 2 above)
Comment (irfen-mi) [reply package.json:11]: yes its material ui libraries. when you want to use material ui components you must install it.

**Comment by kamalbuilds** [comment]
yes I know @developer-irfan but if that would be the case I wont be able to merge your PR  
either fix this or make a fresh PR as there are little changes to be done in a new branch

File: src/components/Navbar.js (hunk 1 above)
Comment (kamalbuilds) [reply src/components/Navbar.js:33]: in the ss its not visible

File: package.json (hunk 2 above)
Comment (kamalbuilds) [reply package.json:11]: :+1:

**Comment by irfen-mi** [comment]
> yes I know @developer-irfan but if that would be the case I wont be able to merge your PR either fix this or make a fresh PR as there are little changes to be done in a new branch

This issue solved in my machine by just running mentioned command. If you want more changes to be done then share the details of that changes.

**Comment by kamalbuilds** [comment]
ok just open a new PR or do changes in this one by deleting the package-lock.json file , I will resolve and merge all the conflicts.

**Comment by kamalbuilds** [comment]
@developer-irfan any update?

**Comment by irfen-mi** [comment]
> @developer-irfan any update?

check now i recently update conflict files like navbar or package.lock.json etc

**Comment by kamalbuilds** [comment]
@developer-irfan we still have many conflicts :eye:

**Comment by kamalbuilds** [comment]
I am closing this for now as its not deploying and has many undesirable conflicts, 
If you want you can make some changes by just copy paste and file a new PR :100:

**Comment by kamalbuilds** [comment]
checking by @Irfo12 are you still working on improving this?

**Pull request closed without merging**